ext/redhat/logrotate

```diff
@@ -1,9 +1,23 @@
-/var/log/puppetlabs/puppet.log {
+# Generic logrotate config for Red Hat systems. Works with both official
+# Fedora and RHEL packages as well as PuppetLabs distributions.
+
+/var/log/puppetlabs/masterhttp.log /var/log/puppet/masterhttp.log {
+  compress
   missingok
   notifempty
-  create 0644 puppet puppet
+  # permissions/owner must be aligned with puppet/defaults.rb
+  create 0660 puppet puppet
```

Comment:
Actually deleting `create` statement is sufficient. Logrotate will inherit from existing logs. More than that, if puppet is able to create missing log files automatically (even via SIGUSR1/2) then I can add `nocreate`.